Yoast/wordpress-seo
# Change @id for Person globally

## Summary

This PR can be summarized in the following changelog entry:

* Schema: fixed some cases where the `@id` for `Person` could be wrong.

## Relevant technical choices:

* Introduced a new `WPSEO_Schema_Utils` utility class.
* Decided not to output the username but use a hash instead, to not expose usernames.

## Test instructions

This PR can be tested by following these steps:

* Make sure the `@id` links to `Person` (for instance from `author`) are correct in all use cases: when site is set to represent company, and when site is set to represent a person, independent of whether the author is also the person representing the site.

## Quality assurance


Fixes #13094
